### DISCUSSION TITLE
Do you use multiple authors?

### DISCUSSION THREAD

**Discussion by estruyf** [estruyf/vscode-front-matter#67] [General]
Are you using the extension as a single user, or do you have multiple authors?

**Comment by Fasani** [comment]
No

**Comment by bwklein** [comment]
Yes, we have many authors/editors for our content.  In one site 'author' is a multiple choice field that is used to pick which authors contributed to an article in the blog. https://github.com/thegooddocsproject/website-hugo/blob/main/frontmatter.json#L68

This drives other Hugo partials in single and list pages to pull information from the Author page and show it in the rendered page. https://github.com/thegooddocsproject/website-hugo/blob/main/layouts/partials/single-partials/author-follow.html

Data file field type wasn't an option when I set that up.  For now, we just add new contributors to the choice field, and make a new markdown file for them.  Ex. https://github.com/thegooddocsproject/website-hugo/blob/main/content/en/author/ryan_macklin.md

The authors use GitPod to edit the site.

**Comment by lorezyra** [comment]
I have multiple authors on my site.

**Comment by michaeltlombardi** [comment]
My projects are a mix - for some I'm the only author, for others there's multiple authors, for others (including the hugo theme framework I'm building) there's not only authors but other contributors - editors, artists, designers, etc.

**Comment by davidsneighbour** [comment]
I use the extension to let users using my theme manage their content. This might be single users or groups of users. So yes, multiple authors. And no, a single author. It depends.

**Reply by michaeltlombardi** [reply]
Color me extremely curious about how you're distributing your Front Matter config defaults/etc to your theme users, since I'm walking down that same road 👀

**Reply by estruyf** [reply]
I hope many good ideas will come from this thread 😁

**Reply by davidsneighbour** [reply]
A public sample can be found on my repo [here](https://github.com/davidsneighbour/kollitsch.dev/tree/main/assets/config/frontmatter). Running `node build.ts` will create the frontmatter.json. 

The idea is to somehow prepare "snippets" and include them via configuration flags. Currently it's just copying this folder around between the projects and adapting to the local needs.

**Reply by michaeltlombardi** [reply]
👀 That's a pretty interesting methodology! You might be interested in the conversation around #407, which is around making base configs for users to override/extend. I came at it similarly from a theme/toolkit maintainer's perspective.
